### PR TITLE
Add/cli endpoint

### PR DESCRIPTION
**Pull request by vsn-hub**
endpoint for cli tool to get client file

### CODE REVIEW

File: internal/endpoints/cli.go [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+		return nil
+	}
+
+	u := usecase.NewIOI(new(cliInput), new(cliOutput), handler)
+	u.SetTitle("OpenApiFile")
+	u.SetDescription("Recieves the open-api file from client and adds to s3")
+	u.SetExpectedErrors(status.InvalidArgument)
+
+	service.Post("/cli", u)
```
Comment (The-TT-Hacker) [inline internal/endpoints/cli.go]: i would call this openapi or something fuzz

File: internal/endpoints/cli.go [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+		if ierr != nil {
+			logger.Infof("There was an issue uploading to s3: %s", ierr.Error())
+		}
+
+		return nil
+	}
+
+	u := usecase.NewIOI(new(cliInput), new(cliOutput), handler)
+	u.SetTitle("OpenApiFile")
```
Comment (The-TT-Hacker) [inline internal/endpoints/cli.go]: should be something like "Open API Fuzz"

File: internal/endpoints/cli.go [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+		sess := session.Must(session.NewSession())
+		uploader := s3manager.NewUploader(sess)
+
+		//will replace with the openapi bucket when created
+		//u := uuid.New()
+		//key := fmt.Sprintf("File/%s", u.String())
+		_, ierr := uploader.Upload(&s3manager.UploadInput{
+			Bucket: aws.String("test-store-swagger"),
+			Key:    aws.String("clientOpenApiFie"),
```
Comment (The-TT-Hacker) [inline internal/endpoints/cli.go]: this should have the uuid in it

File: internal/endpoints/cli.go [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+		bod := strings.NewReader(in.Body)
+
+		sess := session.Must(session.NewSession())
+		uploader := s3manager.NewUploader(sess)
+
+		//will replace with the openapi bucket when created
+		//u := uuid.New()
+		//key := fmt.Sprintf("File/%s", u.String())
+		_, ierr := uploader.Upload(&s3manager.UploadInput{
+			Bucket: aws.String("test-store-swagger"),
```
Comment (The-TT-Hacker) [inline internal/endpoints/cli.go]: this bucket name should be configured by an environment variable which is set by terraform https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/lambda_function#environment

File: internal/endpoints/cli.go (hunk 3 above)
Comment (vsn-hub) [reply internal/endpoints/cli.go]: yep will do was waiting for the s3

**Comment by thefishua** [comment]
Is the issues described above fixed ?

**Comment by vsn-hub** [comment]
> Is the issues described above fixed ?

all fixed and been tested

**Review by thefishua — APPROVED** [review]
LGTM and passes all tests

**Pull request merged**